eventservice/events.yaml

```diff
@@ -2,7 +2,7 @@ openapi: 3.0.0
 info:
   title: Rokwire Events Building Block API
   description: Events Building Block API Documentation
-  version: 1.14.1
+  version: 1.15.0
```

Comment:
Please update the version in the YAML file as the next deployment of these modules will need the latest version. Thanks.